### DISCUSSION TITLE
99925e46-6a14-452f-a807-1a73b3c4006c

### DISCUSSION THREAD

**Discussion by giscus** [varofla/blog_giscus#16] [General]
# 99925e46-6a14-452f-a807-1a73b3c4006c

nRF Connect SDK에 입문하며 I2C를 이용해 온습도 센서 AM2320을 읽어보았습니다.

https://www.varofla.com/99925e46-6a14-452f-a807-1a73b3c4006c

**Comment by varofla** [comment]
### 24.02.04 추가
EasyDMA를 어떻게 사용하나 궁금해서 찾아보니 i2c 디바이스트리의 compatible를 "nordic,nrf-twi"에서 "nordic,nrf-twim"로 변경하면 되는것 같습니다. 변경한 후 디버깅을 했을 때 호출되는 nrfx api가 twi에서 twim으로 바뀌고, twim api에서 dma를 언급하는걸 보아서는 맞는것 같기는 한데...자세한건 조금 더 SDK에 익숙해진 뒤에 이해가 될 것 같습니다.
